apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V63__new_schema_snf_claims.sql

```diff
@@ -0,0 +1,826 @@
+-- NEW_SCHEMA_SNF_CLAIMS.SQL
+-- flyway migration for SNF_CLAIMS and SNF_CLAIM_LINES tables into
+-- a new schema structure that:
+--   1) changes data type of CLM_ID, CLM_GROUP_ID, BENE_ID to BIGINT.
+--   2) organizes parent claim table (SNF_CLAIMS) such that common
+--      claims data columns are organized at top of table structure.
```

Comment:
there are a few other columns whose datatypes are changing so they should also be mentioned here

Reply:
added more comments and explicitly mentioned those columns that went from NUMERIC to an integral data type.